install_config/configuring_authentication.adoc

```diff
@@ -1391,3 +1391,47 @@ a secret with 32 or 64 bytes.
 <3> Encrypting secret, used to encrypt sessions. Must be 16, 24, or 32
 characters long, to select AES-128, AES-192, or AES-256.
 ====
+
+[[configuring-authentication-security]]
+== Security
+
+Administrators can prevent clients from accessing the API by their `User-Agent`
+header in the new `*userAgentMatching*` configuration setting.
+
+For example, you can deny clients using particular libraries or binaries. This
+example denies the Kubernetes 1.2 client binary, OpenShift Origin 1.1.3
+binary, and the POST and PUT *httpVerbs*:
+
+====
+----
+policyConfig:
+  userAgentMatchingConfig:
+    defaultRejectionMessage: "Your client is too old.  Go to https://example.org to update it."
+    deniedClients:
+    - regex: '\w+/v(?:(?:1\.1\.1)|(?:1\.0\.1)) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v(?:1\.1\.3) \(.+/.+\) openshift/\w{7}'
+    - regex: '\w+/v1\.2\.0 \(.+/.+\) kubernetes/\w{7}'amp`
+    httpVerbs:
```

Comment:
'httpVerbs' should be in the same level as 'regex', instead of as 'deniedClients' (I checked: when it is in the same level as 'deniedClients', 'httpVerbs' does not take effect). (The PR https://github.com/openshift/origin/pull/7888/files for the feature also shows this. In https://github.com/openshift/origin/pull/7888/files#diff-3788994f990386b63387ece5c3b5dc33R340, it shows "HTTPVerbs []string" as same level as "Regex string")